config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
```

Comment:
No results found on staging, it's worth investigating whether they got scheduled in the lab.

Reply:
here is seccomp https://lava.ciplatform.org/scheduler/job/509208

Reply:
OK: https://staging.kernelci.org/test/plan/id/617a62c73145858812a6c45b/

The bpf tests appear to be failing, but I guess this might be expected.  Maybe they should be added to the skiplist.  It's not a blocker for this PR though.